experiments/train_estimator.yaml

```diff
@@ -2,6 +2,9 @@
 
 model: estimator
 
+# Model Files will be saved here
+output-dir: runs/estimator
```

Comment:
We don't set it on purpose, since multiple runs will overwrite each other. But for the use case of a single run, it's understandably clearer.